server/routerlicious/packages/services-utils/src/morganLoggerMiddleware.ts

```diff
@@ -120,6 +120,7 @@ export function jsonMorganLoggerMiddleware(
 				[BaseTelemetryProperties.correlationId]: getCorrelationIdWithHttpFallback(req, res),
 				[CommonProperties.serviceName]: serviceName,
 				[CommonProperties.telemetryGroupName]: "http_requests",
+				[HttpProperties.retryCount]: req.query.retry ?? 0,
```

Comment:
1. Probably need to add `retryCount` to the HttpProperties enum
2. might need to do `Number.parseInt(req.query.retry ?? "0")`

Reply:
I don't think there's too much different. since it will be log as string in properties for my understanding.
But let's make it number type in here.